src/kinesis.ts

```diff
@@ -126,7 +121,7 @@ export class KinesisEventHandler<Event, Context> {
             eventID: uuid(),
             kinesis: {
               partitionKey: uuid(),
-              data: stringifyEvent(e),
```

Comment:
There's no need to a custom stringify method now that we're always parsing as json.